packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -147,12 +157,16 @@ export function cloudflare<
 							{
 								type: 'ESModule',
 								path: runnerPath,
+								contents: fs.readFileSync(
```

Comment:
It's kind of annoying that we have to read the module using `fs`. I saw this in the `vitest-pool-workers` package and was hoping it would work without it but it seems like it's the only way.

Reply:
yeah I agree 😓